recipes/perbase/meta.yaml

```diff
@@ -1,29 +1,42 @@
-{% set version = "0.1.0" %}
-{% set sha256 = "901de4b3d4b627169449d73c3873582db3139425f7675eae99be97659a229db7" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "8617bb9a697dce0e8e912ef574253f3001edf85debda5aed7ff2677097100ff8" %}
 
 package:
   name: perbase
   version: {{ version }}
 
 source:
-  url: https://github.com/sstadick/perbase/archive/v0.1.0.tar.gz
+  url: https://github.com/sstadick/perbase/archive/v0.2.1.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
-  noarch: generic
 
 requirements:
   build:
-    - rust
+    - rust >=1.42
+    - clangdev
+    - {{ compiler('c') }}
```

Comment:
That's a better solution to make the linter happy: https://bioconda.github.io/contributor/linting.html#skipping-persistently-on-a-per-recipe-basis

Reply:
Thanks for the link! After fighting it for a while though, the compilers / what they bring with them are actually needed, so it ended up being needed.

Reply:
OK, cool then!